src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IDynamicInvocationExpression.cs

```diff
@@ -0,0 +1,448 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void DynamicInvocation_DynamicArgument()
+        {
+            string source = @"
+class C
+{
+    void M(C c, dynamic d)
+    {
+        /*<bind>*/c.M2(d)/*</bind>*/;
+    }
+
+    public void M2(int i)
+    {
+    }
+}
+";
+            string expectedOperationTree = @"
+IDynamicInvocationExpression (OperationKind.DynamicInvocationExpression, Type: dynamic) (Syntax: 'c.M2(d)')
+  Expression: IOperation:  (OperationKind.None) (Syntax: 'c.M2')
```

Comment:
`OperationKind.None` here seems to be due to a `BoundMethodGroup` node in the tree - we seem to be losing the IOperation tree for the receiver here ('c' for this example). @heejaechang is going to bring it up for discussion in tomorrow's design meeting.